cypress/support/projectConfig/contact_details_page.ts

```diff
@@ -1,17 +1,22 @@
 export default class manualAccountPageObjects {
   static enterPrimaryEmail(primaryEmail: string) {
+    cy.get('#primaryEmailAddress').clear();
     cy.get('#primaryEmailAddress').should('be.visible').type(primaryEmail);
   }
   static enterSecondaryEmail(secondaryEmail: string) {
+    cy.get('#secondaryEmailAddress').clear();
     cy.get('#secondaryEmailAddress').should('be.visible').type(secondaryEmail);
   }
   static enterMobileTelephone(mobileTelephone: string) {
+    cy.get('#mobileTelephoneNumber').clear();
     cy.get('#mobileTelephoneNumber').should('be.visible').type(mobileTelephone);
   }
   static enterHomeTelephone(homeTelephone: string) {
+    cy.get('#homeTelephoneNumber').clear();
     cy.get('#homeTelephoneNumber').should('be.visible').type(homeTelephone);
   }
-  static enterBusinessTelephone(businessTelephone: string) {
-    cy.get('#businessTelephoneNumber').should('be.visible').type(businessTelephone);
+  static enterWorkTelephone(workTelephone: string) {
+    cy.get('#workTelephoneNumber').clear();
+    cy.get('#workTelephoneNumber').should('be.visible').type(workTelephone);
   }
 }
```

Comment:
POM should not be implemented

Reply:
I haven't refactor framework. the ticket still there in backlog

Reply:
That ticket hasn't yet been completed, but we shouldn't be actively creating new steps that we know will need to be removed and refactored. Going forward we need to use the new approach and not fall back to steps that will be removed.